src/univariate/ksdist.jl

```diff
@@ -1,5 +1,37 @@
+# This program is a Julia port of KolmogorovSmirnovDist.c, which is 
+# distributed under the GNU GPL v3 license.
```

Comment:
Note that GPL3 isn't compatible with the license of this package, which is MIT. So I'm afraid that code cannot live in Distributions.jl.

Reply:
I had a look at the paper to see if there was any pseudo-code which could have been the base for a clean-room implementation, but there does not seem to be anything except for a quick description of the Pomeranz recursion formula.

If this issue becomes a blocker for my current project, I could try to implement it from the primary sources, but I cannot promise anything right now.

I also had a quick look at what was available in other languages:
* R has `ks.test`, but it is GPLv3 as well.
* SciPy is BSD3-licensed and implements `scipy.stats.kstest`. I don't know if its computation of the CDF is any better, but if it is then we could probably use it here.